sqlmigrations/1_initial.go

```diff
@@ -12,6 +12,9 @@ import (
 )
 
 func init() {
+	// Do not look for SQL files
+	migrations.DefaultCollection.DisableSQLAutodiscover(true)
```

Comment:
this indeed is a viable option, but I'd rather place a "fix" in a central place, since there will be more migration files here

Reply:
Not sure what you mean here, I put it in the first migration (under init function). `DefaultCollection` is global variable for the `migration` package, once I set in in one place it will apply for all other places if they are under same package `sqlmigrations` in this case.

I can create a separate file for this if you want, like `0_globals.go` and put it there, but the effect will be the same.

Reply:
yep. seems you are right :-)
thanks !